src/JuliaRunClient.jl

```diff
@@ -301,6 +301,16 @@ function _type_name_query(ctx::Context, path::String, job::JRunClientJob, query:
     parse_resp(resp)
 end
 
+function initialize(num_workers, ctx=Context(), job=self())
```

Comment:
would `initializeCluster` be a better name? and `releaseCluster`

Reply:
ok, will change. But will `cluster` be apt ? I was thinking of `initWorkers` initially ?